// Hata yönetimi için try catch kullanımı

let items;

items.forEach(item => console.log(item));

console.log("hata yonetimi duzgun calisiyor");


let info = "kodluyoruz";
console.log(info);